### PR TITLE
Translate several files to Japanese.

### PR DESCRIPTION
**Pull request by rcsv**

### CODE REVIEW

File: ja/reference/applications-explained.rst [hunk 1]
```diff
@@ -36,13 +36,13 @@ Understanding How Phalcon Applications Work
...
 --------------------------
-:doc:`Phalcon\\Mvc\\Application <../api/Phalcon_Mvc_Application>` を使いたくない場合、上述したコードは以下のように変更できます:
+:doc:`Phalcon\\Mvc\\Application <../api/Phalcon_Mvc_Application>` を使いたくない場合、前述のコードは以下のように変更できます:
 
 .. code-block:: php
 
     <?php
 
-    // 「router」サービスを取得
+	// ルーティングサービスを取得する
```
Comment (SidRoberts) [inline ja/reference/applications-explained.rst]: See also line 143, 150, 170.

**Comment by SidRoberts** [comment]
Don't bother translating ja/api/ as it'll be overwritten when the 3.0.2 is released. Once the indentation is fixed, I'll merge it. :)

File: ja/reference/applications-explained.rst (hunk 1 above)
Comment (rcsv) [inline ja/reference/applications-explained.rst]: I've got it. I'll fix it.

**Comment by SidRoberts** [comment]
Thanks. :)

**Pull request merged**